src/SFA.DAS.EmployerAccounts/Data/UserAccountRepository.cs

```diff
@@ -77,6 +77,15 @@ public Task<User> GetUserByRef(Guid @ref)
             return _db.Value.Users.SingleOrDefaultAsync(u => u.Ref == @ref);
         }
 
+        public async Task<Accounts<Account>> GetAccounts()
+        {
+            return new Accounts<Account>
+            {
+                AccountList = await _db.Value.Accounts.ToListAsync(),
+                AccountsCount = await _db.Value.Accounts.CountAsync()
```

Comment:
Rather than re-hit the database, can you not perform `_db.Value.Accounts.ToListAsync()` outside of the return and then supply the list and the list.Count in the object initializer?